discovery-infra/start_discovery.py

```diff
@@ -297,6 +298,13 @@ def _get_vips_ips(machine_net):
 # Converts params from args to assisted-service cluster params
 def _cluster_create_params():
     ipv4 = args.ipv4 and args.ipv4.lower() in MachineNetwork.YES_VALUES
+    ipv6 = args.ipv6 and args.ipv6.lower() in MachineNetwork.YES_VALUES
+    ntp_source = None
```

Comment:
you don't need this in Python

Reply:
Why not?

Reply:
It's redundant. ntp_source will be assigned either by the `if` or by the `else` branch. There's no point in declaring it or initializing it with `None`.